packages/inventory-insights/src/insights.scss

```diff
@@ -46,7 +46,7 @@
         @include rem('margin-bottom', 10px);
         color: #333;
         word-break: break-all;
-        word-wrap: break-word;
+        word-wrap: break-all;
```

Comment:
This should be
`         word-wrap: break-word; `

Reply:
Ah shoot, I edited wrong file.